<?xml version="1.0" encoding="UTF-8"?>
<launch>
<!-- Q-Learning related nodes: -->
  <arg name="plot_window" default="200"/>

  <node
    pkg="q_learning"
    type="train.py"
    name="q_learning_training"
    output="screen" >
  </node>

  <node
    pkg="simulation_tools"
    type="crash_detector"
    name="crash_detector"
    output="screen" >
  </node>

  <node
    pkg="q_learning"
    type="plotter.py"
    name="q_learning_plotter"
    output="screen" >
    <param name="plot_window" value="$(arg plot_window)"/>
  </node>
</launch>
